src/main/java/nl/knaw/dans/pf/language/emd/EmdRelation.java

```diff
@@ -23,6 +23,7 @@
 import java.util.Map;
 
 import nl.knaw.dans.pf.language.emd.types.BasicIdentifier;
```

Comment:
Shouldn't this import be unused by now?
I only see it used in the lines following https://github.com/DANS-KNAW/easy-emd/pull/3/files#diff-c5c77fe3726df4d5187aab4b0e1012d3R76, but I think those should be `BasicString` now as well, right? (GitHub didn't allow me to put a comment there, so I put it here...)